jobs/check_solvers.ts

```diff
@@ -0,0 +1,40 @@
+import { Community, User } from '../models';
+import { Telegraf } from 'telegraf';
+import { MainContext } from '../bot/start';
+import { logger } from '../logger';
+
+const MAX_MESSAGES = 5; // Number of messages before disabling the community
```

Comment:
It would be advisable to add an environment variable to the .env file, so that it can be adjusted without needing to modify the code. Additionally, please include this variable in the .env-sample file.